### ISSUE TITLE
Add type dependencies for react & react-dom

### ISSUE THREAD

**Issue by Kallu609** [sw-yx/create-react-app-parcel-typescript#4]
![image](https://user-images.githubusercontent.com/26361964/43275385-4f5c515a-910a-11e8-99c5-99f0c25e9dc5.png)

Adding these dependencies: `@types/react`, `@types/react-dom` fixes the issue.

**Comment by sw-yx** [comment]
i've had this issue before, i'm not sure how i fixed it to be honest. will look at it soon. thanks for opening this issue. (pr welcome of course if you wanna try fixing)

**Comment by Kallu609** [comment]
Yeah I'm not too familiar with react/webpack, I'll try to fix it if I some free time. Great project btw.

**Comment by sw-yx** [comment]
thank you very much! i gave a talk on the experience too! https://www.youtube.com/watch?v=Et571vTAtT8 if anything the experience of making it was so damn uplifting the journey was really awesome and transferable to other things i work on.
